### ISSUE TITLE
URI for repository that contains latest youtube-dl version

### ISSUE THREAD

**Issue by seymour-krelborn** [ytdl-org/youtube-dl#21975]
## Checklist

- [x ] I'm asking a question
- [x ] I've looked through the README and FAQ for similar questions
- [x ] I've searched the bugtracker for similar questions including closed ones

## Question

WRITE QUESTION HERE

Greetings,

I am new to Linux, and recently installed, via Oracale's VirtualBox, the Debian distribution:
**vm1@debian-vm1:~$** uname -a
Linux debian-vm1 4.9.0-9-amd64 #1 SMP Debian 4.9.168-1+deb9u4 (2019-07-19) x86_64 GNU/Linux

When I used the bundled Synaptic Package Manager to download and install youtube-dl, it installed version:
2017.05.18.1-1

I was able to manually download and run version: 2019.08.02
But I would rather this be handled by my package manager.

So I want to add whatever URI value is needed, to Synaptic's list of repositories, to have it grab the current version.

I have scoured the faq, forums, duckduckgo.com, to no avail.  I cannot figure out what the URI should be, for my package manager to get the current version of youtube-dl

Please advise.

Thank you.

**Comment by dstftw** [comment]
No such official repository URL exists.

**Comment by seymour-krelborn** [comment]
@dstftw 
How did you determine that no such official repository exists?

I am asking, because it would have saved me a fair amount of time searching for what turned out to be a non-existent target, and it would have also saved me the trouble of posting my question.

Also, since there is no official repository (well, at least not for a current version), then what is the process to have github's documentation updated to reflect this, so that others do not struggle as I did searching for what is not there?

All of this is new to me (installed Linux only a few days ago, and joined github when I needed to post this repository question).  So I apologize if I am missing some obvious point, or not following some protocol for my inquiry.

Lastly, for what it's worth, youtube-dl is a very impressive tool, and I can see that I will soon be able to divorce myself from Windows, in favor of the power and flexibility of Linux.

Thank you.
